src/main/features/response/routes/scanned-response-form.ts

```diff
@@ -0,0 +1,19 @@
+import * as express from 'express'
+import { Paths } from 'response/paths'
+import { ErrorHandling } from 'shared/errorHandling'
+import { ScannedDocumentsClient } from 'documents/scannedDocumentsClient'
+import { Claim } from 'claims/models/claim'
+import { DownloadUtils } from 'utils/downloadUtils'
+
+const scannedDocumentsClient: ScannedDocumentsClient = new ScannedDocumentsClient()
+
+/* tslint:disable:no-default-export */
+export default express.Router()
+  .get(Paths.scannedResponseForm.uri,
+    ErrorHandling.apply(async (req: express.Request, res: express.Response, next: express.NextFunction) => {
+      const { externalId } = req.params
+      const pdf: Buffer = await scannedDocumentsClient.getScannedResponseFormPDF(externalId, res.locals.user.bearerToken)
```

Comment:
I would suggest this controller shouldn't know the ScannedDocumentsClient needs a bearer token specifically, but needs to get the response form for the given claim (external ID) with that _user_'s authorisation. If you agree, refactor the method to take the `User` instead of the `bearerToken` and then the client will know that what it needs from the user is its bearer token.